packages/common/async/src/events.ts

```diff
@@ -73,6 +73,23 @@ export type ListenerOptions = {
  */
 // TODO(burdon): Rename EventSink? (Event is a built-in type).
 export class Event<T = void> implements ReadOnlyEvent<T> {
+  /**
+   * Wraps `addEventListener` and return an unsubscription handler.
+   * @param target
+   * @param type
+   * @param listener
+   * @param options
+   */
+  static on<K extends keyof HTMLElementEventMap>(
```

Comment:
consider making this a standalone function

Reply:
+1